models/twilio__account_overview.sql

```diff
@@ -35,12 +45,14 @@ select
     {% endfor %}
     
     count(messages.message_id) as total_messages,
-    round( cast(sum(messages.price) as {{ dbt.type_numeric ()}}), 2) as total_messages_spent,
-    messages.price_unit,
-    round( cast(sum(usage_record.price) as {{ dbt.type_numeric ()}}), 2) as total_account_spent
+    round( cast(sum(messages.price) as {{ dbt.type_numeric ()}}), 2) * -1 as total_messages_spend,
```

Comment:
It read a bit strange with the spend metric for messages being negative. To make it more easily interpreted, I wanted to inverse the totals.